### ISSUE TITLE
Custom icon not shown when using Chrome without isolated profile

### ISSUE THREAD

**Issue by ralienpp** [linuxmint/webapp-manager#59]
I just gave a try to Web Apps and I like the concept. I noticed one minor issue:

- when I use `Chrome`
- and the `isolated profile` feature is *not* enabled

Then the tray icon uses Chrome's icon, rather than the icon I chose in the site's settings. Here is an illustration:

![image](https://user-images.githubusercontent.com/295338/104452091-cd28d400-55a2-11eb-9016-4304c9da45e7.png)

![image](https://user-images.githubusercontent.com/295338/104452233-fc3f4580-55a2-11eb-9e90-251f15ea5ba6.png)

**Comment by Coeur-Noir** [comment]
Use _isolated profile_ to fix this.

**Comment by JakubRimal** [comment]
That is not a fix, it's a workaround.

The point of not using an isolated profile is that you are already logged in (from your "normal" Chrome instance) and it uses all your settings and even extensions of your Chrome. And I guess that's why there is the option to not use the isolated profile.

**Comment by Coeur-Noir** [comment]
You may install any extension you need in an isolated profile, too. And logs it to any account eventually.

Note that while using Firefox, you won't chose isolated or not, it's always isolated.

**Comment by JakubRimal** [comment]
Of course, I understand that but I'm just trying to say that the problem with icon is a bug.

**Comment by mtwebster** [comment]
I investigated this a while back and it appears to be a limitation or bug (depending on your perspective) in chromium itself. It will use the user-specified icon *only* if chromium isn't already running, but then if you open a generic chromium window, it will continue using that app-specific icon.

I haven't come up with any fix or workaround for this, other than using a unique profile, as mentioned above.

**Comment by kolayne** [comment]
@mtwebster (or everyone), is there a chromium bug reported? Could you, please, share a link?
